### ISSUE TITLE
User defined stopwords

### ISSUE THREAD

**Issue by yukiomoto** [valeriansaliou/sonic#300]
Hi,

It is really great that Sonic has built-in stopword lists but this time I want to apply my own stopword list.

I think it would be nice if you could override the build-in stopword lists through the config file or something so Sonic could fit more users' needs. I looked at [CONFIGURATION.md](https://github.com/valeriansaliou/sonic/blob/master/CONFIGURATION.md) but couldn't find a way to do that.

Do you think Sonic could have this feature ?

**Comment by pleshevskiy** [comment]
I agree! I miss it too and now I have to build a server with my patches.

**Comment by RemiBardon** [comment]
Hello @yukidukie and @pleshevskiy,

I will fix this in a minute, it will be released in the next minor version of Sonic 🙂

I will add `stopwords.allow` and `stopwords.deny` to the configuration file.

Also note that in `v2` we will likely remove hardcoded stopwords entirely, as they were a big source of issues in Sonic (in terms of results quality). However if you need to ban a token you’ll still have `stopwords.deny` to do it 👍🏻

**Comment by yukidukie** [comment]
Thank you, @RemiBardon. That sounds very nice !
